runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -120,6 +148,7 @@
 	private int lastAnimation;
 	private boolean loggedInRace;
 	private boolean widgetHiddenChangedOnPvpWorld;
+	private Item[] prevItems = null;
```

Comment:
`null` is the initial value of object fields; it's not necessary to declare it explicitly.
```suggestion
	private Item[] prevItems;
```